tests/e2e/hcp_external_cluster_backup_restore_suite_test.go

```diff
@@ -0,0 +1,93 @@
+package e2e_test
```

Comment:
As this file is still under development and this is part of a sequence of PRs (and not being part of the current E2E execution), I will give you the freedom to continue in a follow up PR with the validations. I'm being a bit more strict with the in-use components.

Reply:
Thanks. Even though the PR title says "full backup/restore" test suite, the main goal of this PR is to setup the way to add validations for the hosted/guest cluster. It adds one basic validation and the point is that we will be adding more in the future (possibly as we add more feature to the hypershift plugin). If you have ideas now on what to validate in the guest cluster, I can follow-up with those changes in the next PR.